src/controllers/users.controller.ts

```diff
@@ -34,11 +35,12 @@ export class UserController {
   }
 
   @Post()
-  async post(@Body() userDTO: SignUpDTO): Promise<InsertResult> {
+  async post(@Body() userDTO: SignUpDTO): Promise<any> {
```

Comment:
Is not a good practice to use any, as someone who comes to the project and tries to read your API will never know what kind of object that endpoint will return. As far I could see, that endpoint returns an instance of a User object, right? Maybe declaring something like "Promise<User>" could fit better.

Reply:
fixed